src/Files.App/Utils/StatusCenter/StatusCenterItem.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Microsoft.UI.Xaml.Controls;
+using System.Windows.Input;
+
+namespace Files.App.Utils.StatusCenter
+{
+	/// <summary>
+	/// Represents an item for StatusCenter.
+	/// </summary>
+	public class StatusCenterItem : ObservableObject
+	{
+		private readonly float _initialProgress = 0.0f;
+
+		private string? _FullTitle;
+		public string? FullTitle
+		{
+			get => _FullTitle;
+			set => SetProperty(ref _FullTitle, value ?? string.Empty);
+		}
```

Comment:
I don't get why you are assigning `string.Empty` when `value is null` even if you marked the property as nullable. I think you can initialize it to `string.Empty` and make it not nullable

Reply:
Actually this is what this was. I will change it.